nixos/modules/virtualisation/xen-dom0.nix

```diff
@@ -692,17 +748,19 @@ in
       '';
 
       # Xen Bootspec extension. This extension allows NixOS bootloaders to
-      # fetch the `xen.efi` path and access the `cfg.bootParams` option.
+      # fetch the dom0 kernel paths and access the `cfg.boot.params` option.
       bootspec.extensions = {
         "org.xenproject.bootspec.v1" = {
-          xen = cfg.efi.path;
-          xenParams = cfg.bootParams;
+          efiPath = cfg.boot.efi.path;
```

Comment:
This breaks lanzaboote, as it relies on the old ("xen") name.

This is an out of tree package, and I'll update it to support new name, but might be annoying...

Reply:
Also, when switching to older generation, older bootloader installation script is unable to create xen entry for the new generations, because they don't have the `xen` field.